internal/formats/spdx22json/decoder_test.go

```diff
@@ -0,0 +1,105 @@
+package spdx22json
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/anchore/syft/syft/pkg"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSPDXJSONDecoder(t *testing.T) {
+	tests := []struct {
+		path          string
+		fail          bool
+		packages      []string
+		relationships []string
+	}{
+		{
+			path:          "alpine-3.10.syft.spdx.json",
+			packages:      []string{"busybox", "libssl1.1", "ssl_client"},
+			relationships: []string{"busybox", "busybox", "libssl1.1", "libssl1.1"},
+		},
+		{
+			path:          "alpine-3.10.vendor.spdx.json",
+			packages:      []string{"alpine", "busybox", "ssl_client"},
+			relationships: []string{},
+		},
+		{
```

Comment:
it looks like a few of these test fixtures assert we find no packages, but also decode correctly. Was it intended to find packages in these tests (outside of the `fail: true` cases)?

Reply:
It was not intended to find packages but rather ensure that the SPDX parsed properly. Maybe this should actually end up as an error if we don't find any packages due to the data present?

Reply:
> It was not intended to find packages but rather ensure that the SPDX parsed properly.

gotcha, that makes sense 👍 

> Maybe this should actually end up as an error if we don't find any packages due to the data present?

no, I think what you have here already is better; we shouldn't require the existence of packages from SBOMs